src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineInstance.java

```diff
@@ -93,7 +97,7 @@ public long getLaunchTimeoutMillis() {
         return launchTimeout * 1000L;
     }
 
-    public ComputeEngineCloud getCloud() throws CloudNotFoundException {
+    public ComputeEngineCloud getCloud() {
```

Comment:
It's still is worthwhile to have the throws statement here, so we can add a javadoc header with a @throws annotation explaining the cause of a potential error.

Reply:
Im not sure if its worth it.. but ok, restored..